### PR TITLE
Update README to include rtc and rendering sample

### LINKED ISSUE #61: Update README to include rtc sample.
### Problem Statement

ReadMe does not have a documentation on the rtc sample or the rendering sample.

### Proposed solution:

Add a one line description bullet e.g:
* blockly-requirejs-sample: Loads RequireJS from a CDN and loads Blockly using AMD.

for the rtc and rendering samples. It should be placed in the top section of the readme.

### PR DESCRIPTION
**Pull request by navilperez**
One liner description of the blockly-rtc and rendering walkthrough sample.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ Sample projects showing how to integrate Blockly into your project.
 - [``blockly-react-sample``](blockly-react/README.md): Blockly in a React project, defines a React Blockly Component.
 - [``blockly-svelte-sample``](blockly-svelte/README.md): Blockly in a Svelte project, defines a Svelte Blockly Component.
 - [``blockly-vue-sample``](blockly-vue/README.md): Blockly in a Vue project, defines a Vue Blockly Component.
+- [``blockly-rtc-sample``](blockly-rtc/README.md): Realtime collaboration environment on top of the Blockly framework.
```
Comment (samelhusseini) [inline README.md]: I don't think this belongs in this section. The readme reads "samples showing how to integrate Blockly into your project", which is not quite what the rtc sample is.

We should create a separate list for listing these kinds of samples.

Comment (navilperez) [reply README.md]: Would the rendering walkthrough sample go under the "samples showing how to integrate Blockly into your project" heading or a new one with the rtc sample?

**Comment by rachel-fenichel** [comment]
@samelhusseini to finish up and merge.

**Comment by samelhusseini** [comment]
@googlebot I consent

**Pull request merged**